src/App.tsx

```diff
@@ -39,7 +39,7 @@ const App = () => {
 	// Prev Page feature
 	const [prevIndex, setPrevIndex] = useState<number>(index);
 	// Renders the selected day's week for the weekly modal
-	const [week, setWeek] = useState<WeekDay[]>(generateWeekList(objDaySelected));
+	const [week, setWeek] = useState<WeekDay[]>(generateWeekList(generateCurrentDateObject()));
```

Comment:
Generating new DateData object on initialization because objDaySelected kept passing in a broken object. Both should work now, will test more later.